instaham/config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 68e402db92ebd00cb3232cf59cc5f35b7cc2c1719b1921eb2aeb1afac506ae7fecc13ce37cab1a550ed2213aa7266be2315c620f9ef89396fcde734746b364b0
+
+test:
+  secret_key_base: 0e4018321915f9c562c3bbe22d7e6c8922ae50debb44279035b8d5fe859f4827426c7468b76fb0778538c45b6a6129b1a0d673ba75dcdca64bb44ec4990ab482
```

Comment:
Shouldn't be here :( b/c others could do the thing with your stuff!!

Please remove these ASAP